framework/src/fwk_multi_thread.c

```diff
@@ -252,6 +252,7 @@ static int put_event(
     }
 
     if (std_event != NULL && std_event->is_delayed_response) {
+#ifdef BUILD_HAS_NOTIFICATION
```

Comment:
unrelated to notifications

Reply:
This is an old commit that has been done when starting to work op-tee architecture without notification. But this doesn't seem necessary anymore. Will remove it